.kokoro/publish_javadoc11.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+set -eov pipefail
+
+if [[ -z "${CREDENTIALS}" ]]; then
+  CREDENTIALS=${KOKORO_KEYSTORE_DIR}/73713_docuploader_service_account
+fi
+
+if [[ -z "${STAGING_BUCKET_V2}" ]]; then
+  echo "Need to set STAGING_BUCKET_V2 environment variable"
+  exit 1
+fi
+
+# switch to java 11
+sudo update-java-alternatives --set java-1.11.0-openjdk-amd64
+$JAVA_HOME/bin/javac -version
+export JAVA_HOME=/usr/lib/jvm/java-1.11.0-openjdk-amd64
+$JAVA_HOME/bin/javac -version
+
+pyenv global 3.7.2
+
+# install pandoc for combining md files
+sudo apt install -q -y pandoc
+
+# Install docuploader package
+python3 -m pip install --upgrade six
+python3 -m pip install --upgrade protobuf
+python3 -m pip install gcp-docuploader==0.6.2
+
```

Comment:
The public_javadoc scripts in googleapis/ were recently migrated to use a requirements.txt file with hash checking to securely install dependencies. Maybe we could do the same here? Here's an example: https://github.com/googleapis/synthtool/pull/1588

Reply:
updated with changes. Test run succeeded.
![image](https://user-images.githubusercontent.com/3340152/200935258-f46c19e9-f865-45c5-adf3-101985a03acc.png)

Reply:
Nice!